lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+  <%%= f.error_notification %>
+
+  <div class="form-inputs">
+  <%- attributes.each do |attribute| -%>
+    <%%= f.<%= attribute.reference? ? :association : :input %> :<%= attribute.name %> %>
```

Comment:
I know you didn't write this but it's weird. A) we don't use turnary operators, only if/else statements, and B) it shouldn't have two `%` signs.